### DISCUSSION TITLE
How do I delete an element from playlist? v0.14.0

### DISCUSSION THREAD

**Discussion by ghost** [libre-tube/LibreTube#3608] [Q&A] [answered]
I don't see the option to remove a video from a playlist. I only find an option to delete the whole playlist.

**Comment by Bnyro** [comment, marked as answer]
By swiping to the right.

**Comment by ghost** [comment]
That was fast!! Thank you!! I think it is to the left though. I didn't even try swiping... It's kind of a standard nowadays, on email clients for example... I shoud've guessed. Sorry.
